src/main/java/org/prebid/server/spring/config/SettingsConfiguration.java

```diff
@@ -179,6 +180,49 @@ HttpApplicationSettings httpApplicationSettings(
         }
     }
 
+    @Configuration
+    @ConditionalOnProperty(prefix = "settings.in-memory-cache.http-update",
+            name = {"endpoint", "refresh-rate", "timeout"})
+    static class HttpPeriodicRefreshServiceConfiguration {
+
+        @Autowired
+        SettingsCache settingsCache;
+        @Value("${settings.in-memory-cache.http-update.endpoint}")
+        String endpoint;
+        @Value("${settings.in-memory-cache.http-update.amp-endpoint}")
+        String ampEndpoint;
+        @Value("${settings.in-memory-cache.http-update.refresh-rate}")
+        long refreshPeriod;
+        @Value("${settings.in-memory-cache.http-update.timeout}")
+        long timeout;
+        @Autowired
+        Vertx vertx;
+        @Autowired
+        HttpClient httpClient;
+        @Autowired
+        ContextRunner contextRunner;
+
+        //FIXME
+        @Autowired
+        Metrics metrics;
+
+        @PostConstruct
+        public void httpPeriodicRefreshService() {
+
+            final HttpPeriodicRefreshService service = new HttpPeriodicRefreshService(settingsCache, endpoint,
+                    refreshPeriod, timeout, vertx, httpClient);
+
+            final HttpPeriodicRefreshService ampService = new HttpPeriodicRefreshService(settingsCache, ampEndpoint,
```

Comment:
Two instances of the service should be configured with different instances of `SettingsCache` - `settingsCache` and `ampSettingsCache`